AdNetworkSupport/Facebook/FacebookBannerCustomEvent.m

```diff
@@ -135,34 +133,37 @@ - (void)dealloc
 - (void)adView:(FBAdView *)adView didFailWithError:(NSError *)error
 {
     MPLogInfo(@"Facebook banner failed to load with error: %@", error.localizedDescription);
-    [LOG log:AD_ERROR];
+    //[LOG log:AD_ERROR];
     [self.delegate bannerCustomEvent:self didFailToLoadAdWithError:error];
 }
 
 - (void)adViewDidLoad:(FBAdView *)adView
 {
 //    MPLogInfo(@"Facebook banner ad did load");
-  //  [self.LOGGER log:@"AD_LOADED"];
+  
     [LOG log:AD_LOADED];
```

Comment:
I'd rename the variable to something else to not have 2 things of the same name. Maybe `MoPubMediation` - I'm also considering using that name on Android.